build.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+/**
+ * @license
+ * Copyright 2019 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { execSync } = require('child_process');
+
+const dockerhubAccount = 'u32915';
+const imageName = 'balena-multibuild-scripts';
+const goLangArchToAlpineArch = {
+	arm: 'arm32v7',
+	arm64: 'arm64v8',
+	386: 'i386',
+	amd64: 'amd64',
+};
+
+function runCmd(cmd) {
+	console.log('=======================================================================');
+	console.log(cmd);
+	execSync(cmd, {stdio: 'inherit'});
+	console.log('=======================================================================');
+}
+
+function buildAndPush() {
+
+	for (const [goLangArch, alpineArch] of Object.entries(goLangArchToAlpineArch)) {
+		// fullImageName example: 'balena/amd64-balena-multibuild-scripts'
+		const fullImageName = `${dockerhubAccount}/${goLangArch}-${imageName}`;
+		runCmd(`docker build -t ${fullImageName} --build-arg ARCH=${alpineArch} src/build-secrets`);
+		runCmd(`docker push ${fullImageName}`);
+		runCmd(`docker rmi ${fullImageName}`);
```

Comment:
I've used this loop to build and push 4 images to Docker Hub:
* `u32915/arm-balena-multibuild-scripts`
* `u32915/arm64-balena-multibuild-scripts`
* `u32915/amd64-balena-multibuild-scripts`
* `u32915/386-balena-multibuild-scripts`

(Where `u32915` is a personal Docker Hub account that should eventually be replaced with `balena`.)
But perhaps the building and and pushing of images should be achieved in a different way, perhaps via `.resinci.yml` instead of this `build.js` script. I probably need advice from @nazrhom ...